admin/broadleaf-admin-module/src/main/java/org/broadleafcommerce/admin/server/service/AdminCatalogService.java

```diff
@@ -34,6 +36,8 @@ public interface AdminCatalogService {
      */
     public Integer generateSkusFromProduct(Long productId);
 
+     Map<String, Object> generateSkus(Long productId);
```

Comment:
I would expect a javadoc or method name with better explanation why it exists and what's the different from  generateSkusFromProduct